test/e2e/common/sysctl.go

```diff
@@ -121,6 +121,47 @@ var _ = framework.KubeDescribe("Sysctls", func() {
 		Expect(log).To(ContainSubstring("kernel.shm_rmid_forced = 1"))
 	})
 
+	It("should support unsafe sysctls which are actually whitelisted", func() {
```

Comment:
this will still fail, see above about concatenating safeWhitelist and unsafeWhitelist.